src/docs/Foundations/Layout/Grid/code.mdx

```diff
@@ -24,43 +19,24 @@ order: 1
 @import 'layouts/_l.flexy';
 ```
 
-`ml-flexy` is very close to the bootstrap flex grid system.
+### How it works
 
-To create a grid, you need two elements:
+The below example creates four equal-width columns at small, medium, large, and extra large breakpoints using our predefined grid classes. The columns are centered in the page within the parent container.
 
-- A grid: `ml-flexy`
-- Columns: `ml-flexy__col`
+<Preview path="basic" />
 
-```html
-<div class="ml-flexy">
-  <div class="ml-flexy__col">
-    col 1
-  </div>
+#### Container
+`ml-container` defines the overall grid context at the top level and applies the proper margin and our default max-width.  You can also remove this max width, by using the grid modifier class ml--container--fluid.
 
-  <div class="ml-flexy__col">
-    col 2
-  </div>
-</div>
-```
-
-<HintItem>
-  Wrap the grid in a ml-container to set external margins and limit the
-  content's maximum width.
-</HintItem>
 
-```html
-<main class="ml-container">
-  <div class="ml-flexy">
-    ...
-  </div>
-</main>
-```
+#### Columns
+`ml-flexy__col` defines individual columns inside a row. Use as many columns as you’d like out of the possible 12 per row. With Flexbox columns widths are set in percentages relative to their parent element so columns without a spec’d width will automatically be equal. For example, four instances of `ml-flexy__col` will will give you four equal width columns — each 25% of the parent from the small breakpoint on up.
```

Comment:
~~inside a row~~ => inside the grid. _(<= we don't have a "row system" inside Mozaic's Grid)_.

~~Use as many columns as you’d like out of the possible 12 per row.~~ => I think that it's not necessary.